functions/schema.js

```diff
@@ -42,7 +42,12 @@ const REIMBURSEMENT_SCHEMA = {
 };
 
 const VOLUNTEER_SCHEMA = {
+  status: STATUS,
+  email: 'Email Address',
   slackUserID: 'Slack User ID',
+  slackEmail: 'Email Address (from Slack)',
+  slackHandle: 'Slack Handle',
```

Comment:
`slackHandle` is not being used anymore as we discusses on Slack @cubeghost but I left it in the "schema" for now